Tribler/Core/Modules/restapi/trustchain_endpoint.py

```diff
@@ -161,3 +164,69 @@ def render_GET(self, request):
 
         blocks = mc_community.persistence.get_latest_blocks(self.identity.decode("HEX"), limit_blocks)
         return json.dumps({"blocks": [dict(block) for block in blocks]})
+
+
+class TrustchainBootstrapEndpoint(TrustchainBaseEndpoint):
+    """
+    Bootstrap a new identity and transfer some reputation to the new key.
+    """
+
+    def render_GET(self, request):
+        """
+        .. http:get:: /trustchain/bootstrap?up=int&down=int
+
+        A GET request to this endpoint generates a new identity and transfers some reputation to it.
+
+            **Example request**:
+
+            .. sourcecode:: none
+
+                curl -X GET http://localhost:8085/trustchain/bootstrap?up=100000&down=40000
+
+            **Example response**:
+
+            .. sourcecode:: javascript
+
+                {
+                    "private_key" : "NEW_KEY",
+                    "public_key"  : "NEW_PUBLIC_KEY" ,
+                     "transactions" : [
+                         {
+                            "hash": "ab672fd6acc0",
+                            "tx" : {
+                                "up": 100000,
+                                "down": 40000,
+                                "total_up": 100000,
+                                "total_down": 40000,
+                            },
+                            "sequence_number": 0,
+                            "link_public_key": "PC_KEY",
+                            "link_sequence_number": 1210,
+                            "previous_hash": 00000000
+                        }
+                    ]
+                    }
+        """
+        mc_community = self.get_trustchain_community()
+        if not mc_community:
+            request.setResponseCode(http.NOT_FOUND)
+            return json.dumps({"error": "trustchain community not found"})
+
+        statistics = mc_community.get_statistics()
+        up = statistics['total_up']
+        down = statistics['total_down']
+        if 'up' in request.args:
+            try:
+                up = int(request.args['up'][0])
+            except ValueError:
+                up = up
```

Comment:
Please raise an error here when the input is not a valid integer so it's consistent with the code in other endpoints.